lib/identity_job_log_subscriber.rb

```diff
@@ -179,6 +184,10 @@ def trace_id(job)
   def duplicate_cron_error?(ex)
     ex.is_a?(ActiveRecord::RecordNotUnique) && ex.message.include?('(cron_key, cron_at)')
   end
+
+  def should_not_warn?(job, ex)
+    (!job.class.try(:warning_messages)&.include(ex.class.name))
```

Comment:
no need for these outer parens, right?
```suggestion
    !job.class.try(:warning_messages)&.include(ex.class.name)
```